File: JavaScript for Beginners/2  Стандартные объекты/2.3.02.js
/*

В этом задании в нашу функцию testStr первым параметром передается строка (переменная str), а вторым - число (переменная n) . 
Вам нужно вернуть из функции символ строки , порядковый номер которого указан в переданном в функцию числе.

Подсказка: порядковый номер не равен индексу символа в строке. Если есть сомнения - перечитайте предыдущие шаги.

Sample Input:

Ithardlycousinmealways
19

Sample Output:

w

*/

function testStr(str, n) {
    return str[n-1];
}

File: Good, good Python/step 02/2.03/2.3.02.py
'''

Подвиг 2. Допишите текст программы для нахождения минимального значения из пяти введенных целых чисел с выводом результата в консоль с помощью функции print.

Sample Input:

8 11 -5 3 0

Sample Output:

-5

'''

numbers = list(map(int, input().split()))
print(min(numbers))
